app_build_suite/build_steps/helm.py

```diff
@@ -336,6 +338,9 @@ def run(self, config: argparse.Namespace, context: Dict[str, Any]) -> None:
                 assert (
```

Comment:
I think I made this mistake, but we need to rewrite both these `asserts` into `raise` - assert won't be handled by any error handling logic higher in the call stack